<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UniPi Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="2" colspan="2">
        <widget class="QLineEdit" name="color5">
         <property name="text">
          <string>#00D9D9</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QSlider" name="control6">
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="pageStep">
          <number>16</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QLineEdit" name="color6">
         <property name="text">
          <string>#0036D9</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QSlider" name="control7">
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="pageStep">
          <number>16</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QSlider" name="control1">
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="pageStep">
          <number>16</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="2">
        <widget class="QLineEdit" name="color7">
         <property name="text">
          <string>#6C00D9</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QSlider" name="control8">
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="pageStep">
          <number>16</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QLineEdit" name="color3">
         <property name="text">
          <string>#6CD900</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QSlider" name="control4">
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="pageStep">
          <number>16</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QLineEdit" name="color4">
         <property name="text">
          <string>#00D936</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QSlider" name="control3">
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="pageStep">
          <number>16</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QLineEdit" name="color2">
         <property name="text">
          <string>#D9A300</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Active</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Colors</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QSlider" name="control5">
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="pageStep">
          <number>16</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QSlider" name="control2">
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="pageStep">
          <number>16</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QSpinBox" name="activeN">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="8" column="2" colspan="2">
        <widget class="QLineEdit" name="color8">
         <property name="text">
          <string>#D900A3</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Levels</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLineEdit" name="color1">
         <property name="text">
          <string>#D90000</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QCheckBox" name="mark">
         <property name="text">
          <string>Mark</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QDoubleSpinBox" name="decay"/>
       </item>
       <item row="9" column="3">
        <widget class="QPushButton" name="setColor">
         <property name="text">
          <string>Set Color</string>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QPushButton" name="sendAll">
         <property name="text">
          <string>Send All</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
